eslzArm/managementGroupTemplates/policyDefinitions/china/mcPolicies.json

```diff
@@ -273,10 +292,13 @@
         //             "/providers/Microsoft.Authorization/roleDefinitions/b24988ac-6180-42a0-ab88-20f7382dd24c"
         //           ],
         //           "deployment": {
-        //             "location": "chinaeast2",
+        //             "location": "northeurope",
```

Comment:
If the RP is not supported, we should ideally not have included this initially. If we comment it out, can we also update the  location property to use 'chinaeast2'?

Reply:
I think this is in the hope that in the future it can just be un-commented. This is the way the file was structured before this PR, so was just following that logic 👍

Have updated location.